interface/eslint.conf.js

```diff
@@ -20,12 +20,11 @@ module.exports = {
   // gNNN: Line number of the same option in ./eslint.google.js
   rules: {
     'curly': 2,                   // (g081) Always use curly braces.
+    'eqeqeq': [2, 'always'],      // (g086) Never allows '==' or '!=' operators.
```

Comment:
Agree that checking for this is a good idea. Comes up on PRs - nice to have automated :+1: